pkmodel/solution.py

```diff
@@ -7,31 +7,45 @@
 
 
 class Solution:
-    """A Pharmokinetic (PK) model solution
-
-    Parameters
-    ----------
-
-    value: numeric, optional
-        an example paramter
-
-    """
     # def __init__(self, models, protocols, therapeutic_min, therapeutic_max):
     def __init__(self, models, therapeutic_min, therapeutic_max):
-        self.models = models
-        self.therapeutic_min = therapeutic_min
-        self.therapeutic_max = therapeutic_max
-        # self.protocols = protocols
-        self.solutions = {}
+        """Defining the system of differential equations and then 
+            solving them and plotting them.
+
+         Args:
+             models TODO I am not sure what to write here
+             therapeutic_min (float): the minimally required drug in a compartment
+                for the drug to be effective.
+             therapeutic_max (float): the maximally permited mass of drug in a
+                compartment before it becomes toxic.        
+        """
+            self.models = models
+            self.therapeutic_min = therapeutic_min
+            self.therapeutic_max = therapeutic_max
+            # self.protocols = protocols
+            self.solutions = {}
```

Comment:
This block of code is now over-indented. You can fix this and commit directly to this branch and it will update this PR, we don't need a new one.